x-pack/legacy/plugins/apm/public/context/UrlParamsContext/resolveUrlParams.ts

```diff
@@ -65,6 +66,7 @@ export function resolveUrlParams(location: Location, state: TimeUrlParams) {
     sortDirection,
     sortField,
     page: toNumber(page) || 0,
+    pageSize: pageSize ? toNumber(pageSize) : undefined,
```

Comment:
nit: Wouldn't this achieve the same?
```js
pageSize: toNumber(pageSize)
```

Reply:
It's slightly different, `parseInt('', 10) === NaN`. You can't use `toNumber(page) || 0` because that would override local defaults, you need an `undefined` state here.

Reply:
Okay, SGTM